MODULE_ACCEPTANCE_CRITERIA.MD

```diff
@@ -79,6 +81,7 @@ It is also important to note that while this document provides the criteria agai
 * Module descriptor MUST include interface requirements for all consumed APIs (3, 5)
   * _This is not applicable to libraries_
 * Third party dependencies use an Apache 2.0 compatible license (2)
+* In order to ensure reproducible builds, snapshot versions of build-time dependencies should not be referenced.
```

Comment:
I hate this, but 100% support it 🙂

Reply:
> requiring non-snapshots could mean releasing something just for the sake of the TCR

I might reframe that as `releasing [package] in order for it to be used in a module`

I reckon this comes back to the conversation yesterday about how we approach first party packages differently because we control them. That can lead to tighter coupling between the development of the packages and the modules using them

I understand it's overhead to release the package. My preference when developing is to make concurrent changes to the package and the module locally then release the package before the changes to the module go to the mainline. 

I accept that isn't always preferable when there is a need to iterate rapidly with interleaving feedback